### PR TITLE
Add Card changes

### PR DESCRIPTION
**Pull request by rahulcs754**
Card Variation :

Cards with badges
Cards with dismiss button
Card with Shadow
Card with up header
Cards with overlay text
Card with Text and Horizontal
Vertical cards
Please review and thanks in advance

https://user-images.githubusercontent.com/6254980/153056404-f6d4433c-5481-44f5-8916-4bc0a55d8312.mp4

### CODE REVIEW

File: docs/intro/pages/Navigation.html [hunk 1]
```diff
@@ -0,0 +1,331 @@
...
+                </div>
+                
+
+            </div>
+            <!--- left sidebar -->
+
+
+            <!--- Right container -->
+            <div class="right-container p-s">
+
```
Comment (tanishq20) [inline docs/intro/pages/Navigation.html]: Use prettier. Avoid giving extra spaces.

File: docs/intro/pages/list.html [hunk 2]
```diff
@@ -0,0 +1,368 @@
...
+        <div class="document-box m-l">
+            <div class="side-bar ">
+
+                <div class="list-container p-s">
+                    <div class="list-heading bt pb-s f-m">
+                         Get Started
+                    </div>
+                    <ul class="list-topic">
+                        <li>
+                            <a class="text-mute" href="">
```
Comment (tanishq20) [inline docs/intro/pages/list.html:47]: `text-mute` class stands for?

**Review by tanishq20 — COMMENTED** [review]
Suggested small changes.

File: docs/intro/pages/list.html (hunk 2 above)
Comment (rahulcs754) [reply docs/intro/pages/list.html:47]: this is provide color 
gray color

File: docs/intro/pages/Navigation.html (hunk 1 above)
Comment (rahulcs754) [reply docs/intro/pages/Navigation.html]: I try to prettier but it is not working so I do myself

**Review by tanishq20 — APPROVED** [review]

**Review by coldpigli — APPROVED** [review]
Everything looks okay. There are some extra whitespace in the code which can be removed.

**Pull request merged**